pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -935,15 +937,24 @@ func appendStatusCauseForCNIPluginHasNoNetworkName(field *k8sfield.Path, incomin
 	return causes
 }
 
-func validateNetworkHasOnlyOneType(field *k8sfield.Path, cniTypesCount int, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+func validateNetworkHasOnlyOneType(field *k8sfield.Path, network *v1.Network, causes []metav1.StatusCause, idx int) []metav1.StatusCause {
+	cniTypesCount := 0
+	if network.Pod != nil {
+		cniTypesCount++
+	}
+	if network.Multus != nil {
+		cniTypesCount++
+	}
```

Comment:
or maybe something like this:

```suggestion
	podTypeExists := network.Pod != nil
	multusTypeExists := network.Multus != nil
	
	if podTypeExists == multusTypeExists { // !XOR
		return append(causes, metav1.StatusCause{
			Type:    metav1.CauseTypeFieldValueRequired,
			Message: "should have one and only one network type",
			Field:   field.Child("networks").Index(idx).String(),
		})
	}
```

Reply:
I changed this, but kept the two possible error messages.